Sources/Foundation/Bridging.swift

```diff
@@ -226,3 +226,29 @@ internal final class __SwiftValue : NSObject, NSCopying {
 
     override var description: String { String(describing: value) }
 }
+
+// MARK: AnyHashable Bridging
+extension AnyHashable : _ObjectiveCBridgeable {
+    public func _bridgeToObjectiveC() -> NSObject {
+        // This is unprincipled, but pretty much any object we'll encounter in
+        // Swift is NSObject-conforming enough to have -hash and -isEqual:.
+        return unsafeBitCast(base as AnyObject, to: NSObject.self)
```

Comment:
This cannot be right. Objects without the ObjC runtime are _not_ NSObjectProtocol-compliant.

Reply:
In particular, they are almost never NSObjects.